### PR TITLE
[MFTF] Add test coverage  for AdminAnalytics module

### LINKED ISSUE #29500: [Issue] [MFTF] Add test coverage  for AdminAnalytics module
This issue is automatically created based on the existing pull request: magento/magento2#29482: [MFTF] Add test coverage  for AdminAnalytics module

---------
**Preconditions:**

This PR creates test for AdminAnalytics module and check is AnalyticsMetadata present in the page source 

**Steps to reproduce:**

1. Go to Admin->Stores->Configuration->Advanced->Admin->Admin Usage and set **Enable Admin Usage Tracking** to **Yes**;
2. Go to Admin->Dashboard and open page source;

**Actual Result:**

**Expected Result:** :heavy_check_mark: AnalyticsMetadata should be present in the page source

### PR DESCRIPTION
**Pull request by Usik2203**
This PR creates test for AdminAnalytics module and check is AnalyticsMetadata present in page source 

### Resolved issues:
1. [x] resolves magento/magento2#29500: [MFTF] Add test coverage  for AdminAnalytics module

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
Hi @slavvka  
Could I kindly ask you help me with failed tests ? I can't understand reason of it . 
Perhaps you can help me ?
Thanks

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by sidolov** [comment]
@magento create issue

File: app/code/Magento/AdminAnalytics/Test/Mftf/Test/AdminCheckAnalyticsTrackingTest.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <argument name="tags" value="config full_page"/>
+            </actionGroup>
+            <reloadPage stepKey="pageReload"/>
+        </before>
+        <after>
+            <magentoCLI command="config:set admin/usage/enabled 0" stepKey="disableAdminUsageTracking"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <waitForPageLoad stepKey="waitForPageReloaded"/>
```
Comment (rogyar) [inline app/code/Magento/AdminAnalytics/Test/Mftf/Test/AdminCheckAnalyticsTrackingTest.xml:32]: I usually recommend putting such actions to a separate action group. However, in this particular case, I don't think that the newly introduced action group will be reusable somehow. So, we may leave it as it is

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8017](https://jira.corp.magento.com/browse/ENGCOM-8017) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**